app/controllers/ConfirmRemoveWebsiteController.scala

```diff
@@ -0,0 +1,83 @@
+import config.{AppConfig, ErrorHandler}
+import controllers.predicates.{AuthPredicateComponents, InFlightPPOBPredicate}
+import javax.inject.{Inject, Singleton}
+import models.User
+import models.customerInformation.UpdatePPOBSuccess
+import models.errors.ErrorModel
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.VatSubscriptionService
+import utils.LoggerUtil.{logInfo, logWarn}
+import views.html.ConfirmRemoveWebsiteView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ConfirmRemoveWebsiteController @Inject()(val authComps: AuthPredicateComponents,
+                                               val inflightCheck: InFlightPPOBPredicate,
+                                               override val mcc: MessagesControllerComponents,
+                                               val errorHandler: ErrorHandler,
+                                               val vatSubscriptionService: VatSubscriptionService,
+                                               confirmRemoveWebsite: ConfirmRemoveWebsiteView,
+                                               implicit val appConfig: AppConfig) extends BaseController(mcc, authComps) {
+
+  implicit val ec: ExecutionContext = mcc.executionContext
+
+  def show: Action[AnyContent] = (blockAgentPredicate andThen inflightCheck).async { implicit user =>
+
+    extractSessionWebsiteAddress(user) match {
+      case Some(website) =>
+        Future.successful(Ok(confirmRemoveWebsite(website)))
+      case _ =>
+        Future.successful(Redirect(controllers.routes.CaptureWebsiteController.show()))
+      }
+  }
+  def removeWebsiteAddress(): Action[AnyContent] = (blockAgentPredicate andThen inflightCheck).async { implicit user =>
+
+    extractSessionWebsiteAddress(user) match {
+      case Some(website) =>
+        vatSubscriptionService.updateWebsite(user.vrn, "") map {
+
+          case Right(UpdatePPOBSuccess(_)) =>
+            //TODO Redirect to the Website address confirmation page
+              Ok("")
+              .removingFromSession(websiteKey, inFlightContactDetailsChangeKey)
+
+          case Left(ErrorModel(CONFLICT, _)) =>
+            logWarn("[ConfirmWebsiteController][updateWebsiteAddress] - There is an website address update request " +
+              "already in progress. Redirecting user to manage-vat overview page.")
+            Redirect(appConfig.manageVatSubscriptionServicePath)
+              .addingToSession(inFlightContactDetailsChangeKey -> "true")
+
+          case Left(_) =>
+            errorHandler.showInternalServerError
+        }
+
+      case _ =>
+        logInfo("[ConfirmWebsiteController][updateWebsiteAddress] - No website address found in session")
+        Future.successful(Redirect(controllers.routes.CaptureWebsiteController.show()))
+    }
+  }
+
+  private[controllers] def extractSessionWebsiteAddress(user: User[AnyContent]): Option[String] = {
+    user.session.get(websiteKey).filter(_.nonEmpty)
```

Comment:
This session key should be `validationWebsiteKey` as that's the one that will get stored when the first page is hit.